examples/sw5/devenv.yaml

```diff
@@ -7,6 +7,11 @@ inputs:
   kellerkinder:
     url: github:kellerkinderDE/devenv-shopware?ref=v2.0.0
     flake: false
+  froshpkgs:
```

Comment:
shopware-cli does not work with shopware 5. I would not install it there 🤔

Reply:
Sounds excellent, but we are using the same devenv-setup for both versions. :-(